### PR TITLE
Create RU ver of becoming_a_qmk_collaborator

### PR DESCRIPTION
**Pull request by greggubarev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
Requesting reviews from @StrikerRUS and/or @nabokovas.

**Comment by greggubarev** [comment]
> @greggubarev Is it machine translation? Could you please humanize this translation?

@StrikerRUS can you give an example - how to translate correctly and in what places?

**Comment by StrikerRUS** [comment]
@greggubarev The whole translation is practically word-to-word one and looks very unnatural. Also, some words have been translated incorrectly and moods in some sentences are wrong. Can you please try to paraphrase the translation, чтобы она выглядела по-русски.

**Comment by greggubarev** [comment]
> @greggubarev The whole translation is practically word-to-word one and looks very unnatural. Also, some words have been translated incorrectly and moods in some sentences are wrong. Can you please try to paraphrase the translation, чтобы она выглядела по-русски 🙂 .

@StrikerRUS обновил

**Comment by StrikerRUS** [comment]
Collaborator - это точно не "сотрудник". Возможно, подойдет "соавтор", или есть какая-то более удачная альтернатива. Может, прямая транслитерация придется к месту...

**Comment by greggubarev** [comment]
> Collaborator - это точно не "сотрудник". Возможно, подойдет "соавтор", или есть какая-то более удачная альтернатива. Может, прямая транслитерация придется к месту...

@StrikerRUS исправил на соавтора

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Стать соавтором QMK
+
+Соавтор QMK - это производитель или разработчик клавиатуры, который заинтересован в том, чтобы помочь QMK вырасти - поддерживая свои клавиатуры, поощряя своих пользователей и клиентов создающих дополнения, идеи и раскладки клавиатур. Мы всегда стремимся добавить больше клавиатур и соавторов, но нам важно, чтобы они соответствовали этим требованиям:
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
Соавтор QMK - это производитель или разработчик клавиатуры, который заинтересован в том, чтобы помочь QMK вырасти. Для этого он полностью поддерживает свои клавиатуры, поощряет пользователей и клиентов создавать новый функционал, идеи и раскладки клавиатур. Мы всегда стремимся добавить больше клавиатур и соавторов, но нам важно, чтобы они соответствовали этим требованиям:
```

Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
Соавтор QMK - это производитель или разработчик клавиатуры, который заинтересован в том, чтобы помочь QMK вырасти - поддерживая свои клавиатуры, поощряя своих пользователей и клиентов создающих дополнения, идеи и раскладки клавиатур. Мы всегда стремимся добавлять больше клавиатур и соавторов, но нам важно, чтобы они соответствовали следующим требованиям:
```

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+* **Иметь PCB, доступный для продажи.** К сожалению, с проводными клавиатурами слишком много вариаций и сложностей.
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
* **Иметь печатную плату на продажу.** К сожалению, у собранных вручную клавиатур слишком много вариаций и сложностей.
```

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+* **Поддерживать вашу клавиатуру в QMK.** Это может просто потребовать начальной настройки, чтобы ваша клавиатура работала, но она также может включать в себя изменения, внесенные в ядро QMK, которые могут сломать или сделать любой пользовательский код избыточным.
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
* **Поддерживать вашу клавиатуру в QMK.** Это может потребовать просто начальной настройки, чтобы ваша клавиатура заработала. Но поддержка также может заключаться в адаптации изменений ядра QMK, которые способны сломать или сделать избыточным любой пользовательский код.
```

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+* **Апрув и слияние пул реквестов по вашей клавиатуре.** Мы хотели бы призвать пользователей предоставлять свои раскладки клавиатур, чтобы другие могли их видеть и работать при создании своих собственных.
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
* **Одобрять и производить слияние пулреквестов для вашей клавиатуры.** Мы хотели бы призвать пользователей предоставлять свои раскладки клавиатур, чтобы другие могли их видеть и работать при создании своих собственных.
```

Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
* **Апрув и слияние пул реквестов по вашей клавиатуре.** Мы призываем пользователей делиться своими раскладками клавиатур, чтобы другие могли их видеть и опираться на них при создании своих собственных.
```

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+Если вы чувствуете, что удовлетворяете этим требованиям, напишите нам письмо по адресу hello@qmk.fm с введением и некоторыми ссылками на клавиатуру!
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
Если вы чувствуете, что удовлетворяете этим требованиям, напишите нам письмо по адресу hello@qmk.fm с кратким описанием и ссылками на вашу клавиатуру!
```

**Comment by greggubarev** [comment]
> Спасибо за перевод! Ниже оставил несколько предложений по улучшению.

@StrikerRUS спасибо, внес изменения!

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Стать соавтором QMK
+
+Соавтор QMK - это производитель или разработчик клавиатуры, который заинтересован в том, чтобы помочь QMK вырасти. Для этого он полностью поддерживает свои клавиатуры, поощряет пользователей и клиентов создавать новый функционал, идеи и раскладки клавиатур. Мы всегда стремимся добавить больше клавиатур и соавторов, но нам важно, чтобы они соответствовали этим требованиям:
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
Соавтор QMK - это производитель или разработчик клавиатуры, который заинтересован в том, чтобы помочь QMK вырасти. Для этого он полностью поддерживает свои клавиатуры, поощряет пользователей и клиентов создавать новый функционал, идеи и раскладки клавиатур. Мы всегда стремимся добавлять больше клавиатур и соавторов, но нам важно, чтобы они соответствовали следующим требованиям:
```

File: docs/ru-ru/becoming_a_qmk_collaborator.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+
+* **Иметь печатную плату на продажу.** К сожалению, у собранных вручную клавиатур слишком много вариаций и сложностей.
+* **Поддерживать вашу клавиатуру в QMK.** Это может потребовать просто начальной настройки, чтобы ваша клавиатура заработала. Но поддержка также может заключаться в адаптации изменений ядра QMK, которые способны сломать или сделать избыточным любой пользовательский код.
+* **Одобрять и производить слияние пулреквестов для вашей клавиатуры.** Мы хотели бы призвать пользователей предоставлять свои раскладки клавиатур, чтобы другие могли их видеть и работать при создании своих собственных.
```
Comment (StrikerRUS) [inline docs/ru-ru/becoming_a_qmk_collaborator.md]:
```suggestion
* **Одобрять и производить слияние пулреквестов для вашей клавиатуры.** Мы призываем пользователей делиться своими раскладками клавиатур, чтобы другие могли их видеть и опираться на них при создании своих собственных.
```

**Comment by greggubarev** [comment]
> Пара потерявшихся комментариев из прошлого ревью:

@StrikerRUS внес, спасибо!

**Comment by greggubarev** [comment]
Guys, can i have feedback about this PR? @noroadsleft @StrikerRUS @nabokovas

**Comment by noroadsleft** [comment]
Спасибо!

**Pull request merged**